src/types/privatekubectl.ts

```diff
@@ -93,23 +106,32 @@ export class PrivateKubectl extends Kubectl {
    }
 
    public extractFilesnames(strToParse: string) {
-      let start = strToParse.indexOf('-filename')
-      let offset = 7
-
-      if (start == -1) {
-         start = strToParse.indexOf('-f')
-
-         if (start == -1) {
-            return ''
+      const fileNames: string[] = []
+      const argv = require('minimist')(strToParse.split(' '))
```

Comment:
we should probably move this import to the top for clarity ex `minimist =  require('minimist')` or ideally an es6 import if it supports that `import * as minimist from 'minimist'`

Reply:
makes sense! changing it to es6 to keep it clean